webui/react/src/components/kit/InputNumber.tsx

```diff
@@ -0,0 +1,18 @@
+import { InputNumber as AntdInputNumber } from 'antd';
+import React from 'react';
+
+interface InputNumberProps {
+  defaultValue?: number;
+  disabled?: boolean;
+  max?: number;
+  min?: number;
+  onChange?: () => void;
+  precision?: number;
+  step?: number;
+  value?: number;
+}
+
+const InputNumber: React.FC<InputNumberProps> = (props: InputNumberProps) => {
```

Comment:
Nitpick but wouldn't `NumberInput` be more natural?

Reply:
InputNumber is what antd calls it, and following that pattern makes all of the Inputs group together in the file explorer, but I don't have strong feelings either way.

Reply:
We don't need to hew to what AntD does